invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/metrics.html

```diff
@@ -8,8 +8,9 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-
+{%  if stats %}
```

Comment:
This `if` looks a bit ugly, is it needed?

Reply:
yes, it fixes broken context